pkgs/os-specific/linux/firmware/raspberrypi/armstubs.nix

```diff
@@ -27,10 +27,10 @@ stdenv.mkDerivation {
     "LD8=${stdenv.cc.targetPrefix}ld"
     "OBJCOPY8=${stdenv.cc.targetPrefix}objcopy"
     "OBJDUMP8=${stdenv.cc.targetPrefix}objdump"
-    "CC=${stdenv.cc.targetPrefix}cc"
-    "LD=${stdenv.cc.targetPrefix}ld"
-    "OBJCOPY=${stdenv.cc.targetPrefix}objcopy"
-    "OBJDUMP=${stdenv.cc.targetPrefix}objdump"
+    "CC7=${stdenv.cc.targetPrefix}cc"
```

Comment:
It also does `-march=armv8-a+crc` above. This does not seem correct on arvm6/armv7.

Reply:
It might, my rpi doesn't boot, but it seems to get beyond the stub. armv7 should have -march=armv7-a